aten/src/ATen/native/README.md

```diff
@@ -50,6 +50,13 @@ signature.
   `Tensor` arguments, these tensors are assumed to be the same type (e.g.,
   if one argument is a `FloatTensor`, all other arguments are checked
   to be `FloatTensor`s.)
+  `Tensor` or `Tensor?` must sometimes be annotated to indicate aliasing and mutability.
+  In general annotations can be defined via the following four situtations
+  `Tensor(a)` - `a` is a set of Tensors that may alias to the same data.
+  `Tensor(a!)` - `a` members of a may be written to thus mutating the underlying data.
+  `Tensor!` - shorthand for Tensor(fresh\_identifier!)
+  `Tensor(a! -> a|b)` - Tensor is in set `a`, written to, and after the write is in set `a` AND `b`.
```

Comment:
based on the explanation here, I'd expect a more full explanation below, but it's not there.  Maybe just remove this example until we have an actual example annotation?